pkg/connectors/protos/register_asset_request.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package connectors;
+option java_package = "com.datmesh";
+option go_package = "github.com/ibm/the-mesh-for-data/pkg/connectors/protobuf";
+
+import "credentials.proto";
+import "data_store.proto";
+
+message RegisterAssetRequest {
+    string dataset_id = 1;           // identifier of the bucket created for registering asset
```

Comment:
I'm not clear on what this is.  The comment says it's the bucket, but isn't the bucket in the DataStore?  The param name is dataset_id, but isn't the dataset_id what is returned after the data set is registered in the catalog?